flows/triggers/base.go

```diff
@@ -133,6 +118,49 @@ func EnsureDynamicGroups(session flows.Session, logEvent flows.EventCallback) {
 	}
 }
 
+//------------------------------------------------------------------------------------------
+// Expressions context
+//------------------------------------------------------------------------------------------
+
+// Context is the schema of trigger objects in the context, across all types
```

Comment:
needed a better way to keep the schema of all triggers consistent in expressions